python-package/lightgbm/dask.py

```diff
@@ -0,0 +1,284 @@
+"""Distributed training with LightGBM and Dask.distributed.
+
+This module enables you to perform distributed training with LightGBM on Dask.Array and Dask.DataFrame collections.
+It is based on dask-xgboost package.
+"""
+import logging
+from collections import defaultdict
+from urllib.parse import urlparse
+
+import numpy as np
+import pandas as pd
+from dask import array as da
+from dask import dataframe as dd
+from dask import delayed
+from dask.distributed import default_client, get_worker, wait
+from toolz import assoc, first
+
+import lightgbm
```

Comment:
can you please remove this `import lightgbm`? Like I mentioned in https://github.com/microsoft/LightGBM/pull/3515#discussion_r523698247, I think it could cause circular-import issues.

It also looks like it might be unnecessary and that you're getting everything you need from `.basic` and `.sklearn` already

Reply:
Done, I've forgot to remove it.

Reply:
no problem, thanks!